pkg/sql/schemachanger/scplan/testdata/create_index

```diff
@@ -6,35 +6,55 @@ ops
 CREATE INDEX id1 on defaultdb.t1(id, name) storing (money)
 ----
 Stage 0
-  *scop.MakeAddedIndexDeleteOnly
-    IndexID: 2
-    IndexName: id1
-    KeyColumnDirections:
-    - 0
-    - 0
-    KeyColumnIDs:
-    - 1
-    - 2
-    SecondaryIndex: true
-    StoreColumnIDs:
-    - 3
-    TableID: 52
+  transitions:
+    from: [SecondaryIndex:{DescID: 52, Name: id1, IndexID: 2}, ABSENT, ADD]
+    to:   [SecondaryIndex:{DescID: 52, Name: id1, IndexID: 2}, DELETE_ONLY, ADD]
```

Comment:
Instead of `from`/`to` could you instead make this output of `transitions` something like:
```
  transitions:
   - [SecondaryIndex:{DescID: 52, Name: id1, IndexID: 2}, ABSENT, ADD] -> DELETE_ONLY
```

Having one line per node would help legibility, the expected output is heavy enough as it is.

Reply:
Done